### PR TITLE
implement board page

### LINKED ISSUE #29: Implement Board Page
image

- Create a data file containing an array of objects. 
- Each object containing name, position, info (should be a string), fact, and img.
- Map through each board members using the Board.jsx component.
- Add Title component
- You may add more columns instead of 2 columns.
- Ignore blue green blobs in the background.

https://acm-ucr.github.io/docusaurus/docs/react/mapping-filtering

### PR DESCRIPTION
**Pull request by xwRongx**
image

### CODE REVIEW

File: src/app/board/page.js [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+import { BOARD } from "@/data/boardMembers";
+import Title from "@/components/Title";
+
 const Page = () => {
-  return <div>Board</div>;
+  return (
+    <div>
+      <div>
+        <Title text="Get To Know Us" />
+        <div className="grid lg:grid-cols-3 md:grid-cols-2 sm:grid-cols-1 gap-y-16 gap-x-20 mx-16 my-16">
```
Comment (seanquiambao) [inline src/app/board/page.js]:
```suggestion
        <div className="grid lg:grid-cols-3 md:grid-cols-2 sm:grid-cols-1 gap-y-16 gap-x-20">
```
don't use margining to position elements.

File: src/app/board/page.js [hunk 2]
```diff
@@ -1,5 +1,27 @@
+import Board from "@/components/board/Board";
+import { BOARD } from "@/data/boardMembers";
+import Title from "@/components/Title";
+
 const Page = () => {
-  return <div>Board</div>;
+  return (
+    <div>
+      <div>
```
Comment (seanquiambao) [inline src/app/board/page.js]: remove the extra div tag

Comment (seanquiambao) [reply src/app/board/page.js]: image

try centering the elements in the parent tag using flexbox and items/justify alignment

Comment (seanquiambao) [reply src/app/board/page.js]: i would also try adding one more column

**Review by seanquiambao — CHANGES_REQUESTED** [review]

File: src/app/board/page.js [hunk 3]
```diff
@@ -1,5 +1,27 @@
+import Board from "@/components/board/Board";
+import { BOARD } from "@/data/boardMembers";
+import Title from "@/components/Title";
+
 const Page = () => {
-  return <div>Board</div>;
+  return (
+    <div className="flex justify-center items-center p-3">
+      <div className="grid grid-cols-1">
+        <Title text="Get To Know Us" />
```
Comment (seanquiambao) [inline src/app/board/page.js]: image

add spacing between the title and cards.

i would also pull changes from dev and changing the colors (based on the Figma file)

**Review by seanquiambao — CHANGES_REQUESTED** [review]
this should be good to go after the comments!

File: src/app/board/page.js (hunk 3 above)
Comment (xwRongx) [reply src/app/board/page.js]: about the changing colors, for the title the tailwind config file does not have the lighter green shown in the figma file. Should I add it as like green-300?

Comment (seanquiambao) [reply src/app/board/page.js]: yes 👍

**Comment by xwRongx** [comment]
image

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**